client/src/components/Applications/AppForm/AppForm.jsx

```diff
@@ -682,6 +736,28 @@ export const AppSchemaForm = ({ app }) => {
                   <div className="appSchema-header">
                     <span>Configuration</span>
                   </div>
+                  {app.execSystems &&
+                    Object.keys(execSystemsWithAllocation).length > 1 && (
+                      <FormField
+                        label="Execution System"
+                        name="execSystemId"
+                        description="Select the system this job will execute on."
+                        type="select"
+                        required
+                      >
+                        {execSystemsWithAllocation
+                          .sort()
+                          .map((exec_system_id) => (
+                            <option key={exec_system_id} value={exec_system_id}>
+                              {getSystemName(
+                                getExecSystemFromId(app, exec_system_id)?.host
+                              )}{' '}
+                              : {exec_system_id} :{' '}
+                              {getExecSystemFromId(app, exec_system_id)?.host}
```

Comment:
Did we come up with an alternative for 
```
Frontera : frontera : frontera.tacc.utexas.edu
```

- Could show only `Frontera`, `LoneStar6`, etc if we know the name, otherwise show systemId and assume user understands host? Or host in a separate, read-only, disabled box?

Reply:
I made the changes now from the discussion. Below are the changes:

* App definition notes will have this note:
```
        "dynamicExecSystems": {
                "frontera": "Frontera",
                "ls6": "Lonestar6"
        }
```
This will have a map of system id to the label.
To use this, the search in system list will now look for specific ids.
* System list is shown even if the size is 1 for dynamic execution list.
* The initial list of execution systems is based on the portal alloc (if available). This has an issue though, lets say a portal (like 3DEM) has portaAlloc but no execution system assigned to it, then user will never be able to execute anything. What is fallback here? if no execution system matches portalAlloc should we populate the execution system list with full list? This change will definitely break 3DEM unless 3DEM allocation model is changed.

* Allocation is moved right next to System list. Help text is adjusted, will ping design channel for feedback on this.
* Just like previously, allocation list is filtered based on what is selected in system list.
* When dynamic execution list is setup but there is no allocation available, then the error message is adjusted for allocation.
*